src/qibolab/oneQ_emulator.py

```diff
@@ -0,0 +1,56 @@
+from qibolab.emulator import create_runcard_emulator
+from qibolab.instruments.simulator.models import general_no_coupler_model
+from qibolab.instruments.simulator.pulse_simulator import (
+    PulseSimulator,
+    get_default_simulation_config,
+)
+
+# device parameters: frequency in Hz, time in s
+device_name = "ibmfakebelemQ0"
+sampling_rate = 4500000000.0
+readout_error = [0.01, 0.02]
+lo_freq = 5090167234.445013
+anharmonicity = -336123005.1821652
+drive_freq = 5090167234.445013
+rabi_freq = 125457538.19061986
+T1 = 8.857848970762537e-05
+T2 = 0.00010679794866226273
+nlevel = 3
+
+# simulation parameters
+sim_sampling_boost = 10
+simulate_dissipation = True
+instant_measurement = True
+
+
+def create_oneQ_emulator(runcard_folder: str):
```

Comment:
Why do you need the `runcard_folder` here since the parameters are hardcoded in the Python script?

Reply:
As mentioned during the meeting, it behaves as a typical platform and so is required to load a runcard for pulse information. The parameters there are for the model that will be emulated, which for more complicated systems, will be stored in a separate json file (perhaps model.json or model_params.json)?

Reply:
If you follow the directory structure proposed above in https://github.com/qiboteam/qibolab/pull/849#discussion_r1538802695 I think you don't need to explicitly have a platform argument. You can hardcode it to look for the json that is in the same path, like we do for other platforms, for example here:
https://github.com/qiboteam/qibolab/blob/abe3698550a84c5bf8e02c64e54afa7f90eb1e67/tests/dummy_qrc/qm/platform.py#L14

Reply:
Fixed in 15307d6